src/utils/hints/oracles.js

```diff
@@ -0,0 +1,6 @@
+export const oraclesHints = {
+  oracleId: 'Identifier of Oracle that is similar to the address of the Account that registered the Oracle. The only difference is the prefix that will be ok_ instead of ak_. This means that each account can only host 1 oracle. It\'s not possible to manage multiple oracles using the same account.',
+  oracleRegistered: 'To register an Oracle you need to provide a queryFormat and a responseFormat to the registerOracle function of the SDK. In addition to the common transaction options you can provide the oracle specific options queryFee and oracleTtlValue',
```

Comment:
I think this description is unrelated to what we show, which is simply the keyblock and the date when the registration happened

Reply:
yes, it is just the moment of time where the oracle has been registered via `OracleRegisterTx`

Reply:
I agree, we should just summarise here what the meaning/implication of registered is

Reply:
changed to
`Date, time, and block height when the Oracle has been registered via OracleRegisterTx.`